provisioning/aws/scripts/deploy.sh

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
-set -Eeuo pipefail
-
-aws eks update-kubeconfig --name "$AWS_EKS_CLUSTER_NAME" --region "$AWS_REGION"
-kubectl config set-context --current --namespace templates-api
```

Comment:
Tento riadok som zmazal, aby to bolo rovnko ako v `azure.sh`.
Namiesto toho som pridal k `kubectl rollout` argument `--namespace templates-api`.

Teraz je layout `aws.sh` a `azure.sh` rovnaky.